engine/sdk/src/test/test_sdk_lib.cpp

```diff
@@ -48,9 +48,9 @@ dmExtension::Result UpdateTest(dmExtension::Params* params)
 
 void OnEventTest(dmExtension::Params* params, const dmExtension::Event* event)
 {
-    if( event->m_Event == dmExtension::EVENT_ID_ACTIVATEAPP )
+    if( event->m_Event == (ExtensionEventID)dmExtension::EVENT_ID_ACTIVATEAPP )
         ++g_TestAppEventCount;
-    else if(event->m_Event == dmExtension::EVENT_ID_DEACTIVATEAPP)
+    else if(event->m_Event == (ExtensionEventID)dmExtension::EVENT_ID_DEACTIVATEAPP)
```

Comment:
I'll also keep the PR release notes updated with the migration notes I collect when testing/fixing the extensions in test-sdk.